pkg/anago/anago.go

```diff
@@ -184,8 +216,7 @@ func (s *Stage) Run() error {
 	parentBranch := ""
```

Comment:
I would like to have the `parentBranch` as part of the states.

Reply:
Perfect, I had thought adding that one as well.